### ISSUE TITLE
Finish Documentation of Database Backends

### ISSUE THREAD

**Issue by gtfierro** [BrickSchema/docs#1]
Current page at: https://docs.brickschema.org/software/database.html

Documentation/links required for:
- [ ] Oxigraph
- [ ] Apache Jena
- [ ] Blazegraph
- [ ] TopBraid

**Comment by linsong** [comment]
Glad  to know there will be a document about the backend databases, this will be pretty helpful. 

I found the [Redland RDF Library](http://librdf.org/)  is useful. It includes a set of libraries/tools for graph storage, RDF data file parsers and SPARQL query(SPARQL 1.1 partly supported, no property path support yet) library. Since it is written in C, a good fit for low end device.  

I tried the [roqet](http://librdf.org/rasqal/roqet.html) command line that supports SPARQL on a TTL file, it works well.

One main drawback is its development had been stopped since 2014, not sure if this is relevant to the new document or not.

**Comment by iamliamc** [comment]
When you get a chance @gtfierro you can checkout my basic implementation using Neo4j and Neosemantics... 
https://github.com/iamliamc/neo4j-brick

**Comment by georghildebrand** [comment]
We recently benchmarked and tried several databases for building knowledge graphs including brick ontology for smart homes but also smart factory etc. One important aspect for us is to have ability 

@gtfierro you could consider adding:

- [ ] Azure digital Twins
- [ ] Stardog
- [ ] Virtuoso

**Comment by gtfierro** [comment]
@georghildebrand it looks like you didn't finish a sentence there ("One important aspect for us to have ability...") --- could you fill in the needs of the platform?

I've added the other platforms to the list of documentation we will need to add. If you are able, it would help me (and the Brick community) out include some of your findings in our documentation -- anything you can contribute would be welcome!

**Comment by georghildebrand** [comment]
Yes true, looks like I went to a meet and then just send out all the open tabs >D. Thx @gtfierro
